From 59dd32d063ca09a64e371768d7fc74a9954aa717 Mon Sep 17 00:00:00 2001
From: William Grant <wgrant@ubuntu.com>
Date: Fri, 9 Jun 2017 09:10:02 +0800
Subject: [PATCH] configure.ac: fix install-sh not found

Fix configure.ac to cope with new autoconf.
Recent autoconfs generate a bad configure when AM_INIT_AUTOMAKE is
called as late as it was, ending up thinking that the am_aux_dir is pwd
at the start of the build. Move it up to under AC_INIT to fix that.

Upstream-Status: Pending

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=724116

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

---
 configure.ac | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 32a76169..d682ad78 100644
--- a/configure.ac
+++ b/configure.ac
@@ -22,6 +22,13 @@ dnl     checks for system services
 AC_INIT([resource-agents], 
 	m4_esyscmd([make/git-version-gen .tarball-version]),
 	[developers@clusterlabs.org])
+AC_CONFIG_AUX_DIR(.)
+
+dnl
+dnl AM_INIT_AUTOMAKE([1.11.1 foreign dist-bzip2 dist-xz])
+dnl
+
+AM_INIT_AUTOMAKE([1.10.1 foreign dist-bzip2])
 
 AC_USE_SYSTEM_EXTENSIONS
 
@@ -29,7 +36,6 @@ CRM_DTD_VERSION="1.0"
 
 PKG_FEATURES=""
 
-AC_CONFIG_AUX_DIR(.)
 AC_CONFIG_MACRO_DIR([m4])
 
 AC_CANONICAL_HOST
@@ -96,12 +102,6 @@ AC_ARG_WITH([systemdtmpfilesdir],
      fi
 AM_CONDITIONAL(HAVE_SYSTEMD, [test -n "$with_systemdtmpfilesdir" -a "x$with_systemdtmpfilesdir" != xno ])
 
-dnl 
-dnl AM_INIT_AUTOMAKE([1.11.1 foreign dist-bzip2 dist-xz])
-dnl
-
-AM_INIT_AUTOMAKE([1.10.1 foreign dist-bzip2])
-
 AC_DEFINE_UNQUOTED(AGENTS_VERSION, "$PACKAGE_VERSION", Current agents version)
 
 CC_IN_CONFIGURE=yes
